workflows/diagnostics/fv3net/diagnostics/offline/compute_diagnostics.py

```diff
@@ -63,8 +62,12 @@ def compute_diagnostics(
     delp: xr.DataArray,
     n_jobs: int = -1,
 ):
+    if "ncol" in grid.area.dims:
+        horizontal_dims = ["ncol"]
+    else:
+        horizontal_dims = ["x", "y", "tile"]
```

Comment:
I suggest inferring the `horizontal_dims` once in the main function of the compute code, and then passing it down to this function. Will be less liable to bugs and easier to make changes if the dim name convention changes for SCREAM.